content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+  :guilabel:`Available` allows other meetings to be scheduled for the same time. It is recommended
+  to set this field to :guilabel:`Busy`, so no other meetings can be scheduled for the attendees at
+  the same time.
+
+.. image:: recruitment/new-event.png
+   :align: center
+   :alt: A new meeting card with all the details populated, and ready to save.
+
+Send meeting to attendees
+*************************
+
+Once changes have been entered and the meeting details are correct, the meeting can be sent to the
+attendees via email or text message.
+
+To send the meeting via email, click the :guilabel:`Email` button next to the list of attendees. A
+:guilabel:`Contact Attendees` pop-up appears. A pre-formatted email using the default
+:guilabel:`Calendar: Event Update` email template populates the email body field. The followers of
+the document (job application), as well as the :guilabel:`Recruiter` are added as
+:guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email
+to the applicant as well. Make any other desired changes to the email. If an attachment is needed,
+click the :guilabel:`Attach A File` button, navigate to the file, then click :guilabel:`Open`. Once
+the email is ready to be sent, click :guilabel:`Send`.
+
+.. image:: recruitment/email-event.png
+   :align: center
+   :alt: Enter the information to send the event via email.
+
+To send the meeting via text message, click the :guilabel:`SMS` button next to the list of
+attendees. A :guilabel:`Send SMS Text Message` pop-up appears. At the top, in a blue box,
+information appears regarding the message. The first line lists how many records (phone numbers) are
+selected. The second line lists the number of recipients and how many of them are invalid. If a
+contact does not have a valid phone number listed, click :guilabel:`Close`, and edit the attendee's
+record, then redo these steps.
+
+.. warning::
+   It is *not* recommended to check the box to send to all the records in the :guilabel:`Send SMS
+   Text Message` pop-up. This would send a message to all contacts with a valid phone number.
+
+When the message reads :guilabel:`0 are invalid`, type in the message to be sent in the
+:guilabel:`Message` field. Beneath the message field, the number of characters, as well as the
+amount of text messages required to send the message (according to GSM7 criteria) appears. Click
+:guilabel:`Put In Queue` to have the text sent later, after any other messages are scheduled, or
+click :guilabel:`Send Now` to send the message immediately.
+
+.. image:: recruitment/send-sms.png
+   :align: center
+   :alt: Send a text message to the attendees of the meeting.
+
+.. note::
+   Sending text messages is not a default capability with Odoo. To send text messages, credits are
+   required, which need to be purchased. For more information on credits and plans, refer to
+   :doc:`this documentation <../marketing/sms_marketing/pricing/pricing_and_faq>`.
```

Comment:
IAP credits -- note

Reply:
reference docs: https://www.odoo.com/documentation/16.0/applications/general/in_app_purchase.html

Reply:
Thank you! I thought the other was the right doc- I was wrong =)!